_posts/hyotester/2022-09-22-hyotester-first-focused-reflection.md

```diff
@@ -1,8 +1,202 @@
 ---
 layout: post
 author: hyotester
-title: "Hyo Tester's First Focused Reflecction"
+title: "Hyo's First Focused Reflection"
 ---
 
-This is my first post!
+# Introduction
+Hi, I'm hyojeong! This posting is my first focused reflection, where I am exploring and focusing on code and how that specific code has helped me learn throughout the class. 
+
+## Let's get started to understand Functional Turtles
+
+September is my birthday!! So I tried to learn how to write the Happy Birthday Python Program during Turtle. Also, I use input function to know that How many friends will come to my birthday party?. 
+
+You may be wondering that who invited to my birthday party. So, I used function to iterate variable name inside loop. This function let you know the people  invited to my birthday party.
+
+<iframe src="https://trinket.io/embed/python/4103681aad" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+#The loop / variable name 
+
+```python
+for name in ["Joe", "Amy", "Brad", "Angelina", "Zuki", "Thandi", "Paris"]:
+    print("Hi", name, "Please come to my party on Saturday!")
+        
+```
+
+# Input 
+```python
+import turtle
+
+window = turtle.Screen()
+window.bgcolor("lightgreen")
+
+shape = turtle.Turtle()
+shape.color("blue")
+shape.pensize(3)
+
+sides = int(input("How many friends will come to my birthday party? "))
+
+def polygon(sides,length):
+
+    for x in range(sides):
+        shape.forward(length)
+        shape.left(360/sides)
+        
+        
+from datetime import datetime
+today = datetime.today()
+age_str = input('What is your age?\n')
+age = int(age_str)
+birth_year = today.year - age
+print("You were born in " + str(birth_year) + " or " + str(birth_year - 1))
+
+```
+
+#Creating a Cake with Topping and a Plate
```

Comment:
Hi Hyo, was this meant to be a header for the next section? I think you might have forgotten a space between "#" and "Creating a Cake with Topping and a Plate"